Run Linters Are Failing On Master

Linters are failing on master and any subsequent PRs, due to something a skew that appears to have occurred between #75164 and #75067: https://github.com/tgstation/tgstation/actions/runs/4908974378/jobs/8765080144#step:7:871

![image](https://user-images.githubusercontent.com/34697715/236700365-23938e1d-b7f6-4c48-8a92-35bc488720f3.png)

```txt
packages/tgui/interfaces/RequestsConsole/MessageWriteTab.tsx(86,13): error TS2322: Type 'unknown[]' is not assignable to type 'string[] | DropdownEntry[]'.
  Type 'unknown[]' is not assignable to type 'string[]'.
    Type 'unknown' is not assignable to type 'string'.
packages/tgui/interfaces/RequestsConsole/MessageWriteTab.tsx(95,13): error TS2322: Type 'unknown[]' is not assignable to type 'string[] | DropdownEntry[]'.
packages/tgui/interfaces/RequestsConsole/MessageWriteTab.tsx(104,13): error TS2322: Type 'unknown[]' is not assignable to type 'string[] | DropdownEntry[]'.
=> Target 'tgui-tsc' failed in 25.915s, exit code: 2
```

Not sure how to fix this because I have skill of noob when it comes to TypeScript stuff like this.

--- a/tgui/packages/tgui/interfaces/RequestsConsole/MessageWriteTab.tsx
+++ b/tgui/packages/tgui/interfaces/RequestsConsole/MessageWriteTab.tsx
@@ -1,5 +1,5 @@
 import { useBackend, useLocalState } from '../../backend';
-import { sortBy } from 'common/collections';
+import { sortStrings } from 'common/collections';
 import { Box, Button, Dropdown, Section, Stack, TextArea } from '../../components';
 import { RequestsData, RequestType, RequestPriority } from './types';
 
@@ -13,9 +13,9 @@ export const MessageWriteTab = (props, context) => {
     information_consoles = [],
   } = data;
 
-  const sorted_assistance = sortBy((console) => console)(assistance_consoles);
-  const sorted_supply = sortBy((console) => console)(supply_consoles);
-  const sorted_information = sortBy((console) => console)(information_consoles);
+  const sorted_assistance = sortStrings(assistance_consoles);
+  const sorted_supply = sortStrings(supply_consoles);
+  const sorted_information = sortStrings(information_consoles);
 
   const resetMessage = () => {
     setMessageText('');
@@ -83,7 +83,7 @@ export const MessageWriteTab = (props, context) => {
         {requestType === RequestType.ASSISTANCE && (
           <Dropdown
             width="100%"
-            options={sorted_assistance.map((recipient) => recipient)}
+            options={sorted_assistance}
             selected={recipient}
             displayText={recipient || 'Pick a Recipient'}
             onSelected={(value) => setRecipient(value)}
@@ -92,7 +92,7 @@ export const MessageWriteTab = (props, context) => {
         {requestType === RequestType.SUPPLIES && (
           <Dropdown
             width="100%"
-            options={sorted_supply.map((recipient) => recipient)}
+            options={sorted_supply}
             selected={recipient}
             displayText={recipient || 'Pick a Recipient'}
             onSelected={(value) => setRecipient(value)}
@@ -101,7 +101,7 @@ export const MessageWriteTab = (props, context) => {
         {requestType === RequestType.INFORMATION && (
           <Dropdown
             width="100%"
-            options={sorted_information.map((recipient) => recipient)}
+            options={sorted_information}
             selected={recipient}
             displayText={recipient || 'Pick a Recipient'}
             onSelected={(value) => setRecipient(value)}

